src/scripts/pokemons/PokemonList.ts

```diff
@@ -12725,7 +12725,7 @@ const pokemonList: PokemonListData[] =
         },
         {
             'id': 648,
-            'name': 'Meloetta',
+            'name': 'Meloetta (aria)',
```

Comment:
I'm not able to comment on the other form because it's not near any changes, but it looks like the form name should be lowercased there as well:
https://github.com/pokeclicker/pokeclicker/blob/3c5462a8b1d8f0766ba83dd742b9a09fd8c8dcec/src/scripts/pokemons/PokemonList.ts#L12745
 (Pirouette) ->  (pirouette)